packages/blade/src/components/Table/docs/BasicTable.stories.tsx

```diff
@@ -151,6 +184,18 @@ const TableTemplate: StoryFn<typeof TableComponent> = ({ ...args }) => {
                 </TableRow>
               ))}
             </TableBody>
+            <TableFooter>
+              <TableFooterRow>
+                <TableFooterCell>Total</TableFooterCell>
+                <TableFooterCell>-</TableFooterCell>
+                <TableFooterCell>-</TableFooterCell>
+                <TableFooterCell>-</TableFooterCell>
```

Comment:
You might have to add another TableFooterCell here ifselectionType is multiple. Otherwise there will be a blank cell

Reply:
Added